pxr/imaging/plugin/hdRpr/rprApi.cpp

```diff
@@ -3377,9 +3375,7 @@ Don't show this message again?
 
 private:
     static RprUsdPluginType GetPluginType(TfToken const& renderQuality) {
-        if (renderQuality == HdRprCoreRenderQualityTokens->Full) {
```

Comment:
fter remove "Full" setting from generateRenderSettingFiles.py token HdRprCoreRenderQualityTokens->Full is unavailable